deep_water_level/src/deep_water_level/infer.py

```diff
@@ -37,7 +37,7 @@ def load_model(model_path: Path, train_water_line: bool = False):
 
     checkpoint = torch.load(model_path, weights_only=False, map_location=my_device())
     model_args = checkpoint["model_args"]
-    preprocessing_args = checkpoint["preprocessing_args"] if "preprocessing_args" in checkpoint else None
+    preprocessing_args = checkpoint["preprocessing_args"] if "preprocessing_args" in checkpoint else {}
```

Comment:
FYI, you can do this more concisely doing `checkpoint.get("preprocessing_args", {}`